src/index.js

```diff
@@ -11,10 +11,35 @@ const buildImport = template(`
       if (ns && ns.__esModule) {
         resolve(ns);
       } else {
-        var wrap = Object.create(null);
-        wrap.__esModule = true;
-        wrap.default = ns;
-        Object.freeze(wrap);
+        var wrap;
+        if (typeof Object.create === 'function') {
+          wrap = Object.create(null);
+        } else {
+          wrap = {};
+          if (typeof wrap.__proto__ === 'object') {
+            wrap.__proto__ = null;
+          }
+        }
+        if (typeof Object.defineProperty === 'function') {
+          Object.defineProperty(wrap, '__esModule', {
+            value: true,
+            writable: false,
+            enumerable: false,
+            configurable: false
+          });
+          Object.defineProperty(wrap, 'default', {
```

Comment:
i think we can assign `default` unconditionally, since anything with `defineProperty` will also have `freeze`, and `[[Put]]` + freeze results in this descriptor.